.github/workflows/feature_deploy.yml

```diff
@@ -48,16 +49,16 @@ jobs:
     # environment:
     #   name: ${{ needs.deployment-environment.outputs.environment }}
     strategy:
-      max-parallel: 4
+      max-parallel: 2
       matrix:
         repository:
-          - portal-ui
-          - portal-api
-          - gef-ui
-          - trade-finance-manager-ui
-          - trade-finance-manager-api
+          # - portal-ui
```

Comment:
Not entirely sure what this is doing but I'm not too familar with github actions yaml